TypeCobol.Test/Parser/Programs/Cobol85/ExecSql/ExecSqlWithTruncateStatement.rdz.cbl

```diff
@@ -0,0 +1,22 @@
+       IDENTIFICATION DIVISION.
+       PROGRAM-ID. DVZZMFT3.
+       DATA DIVISION.
+       WORKING-STORAGE SECTION.
+             EXEC SQL 
+              TRUNCATE TABLE INVENTORY
+              REUSE STORAGE
+              IGNORE DELETE TRIGGERS
+              TRUNCATE TABLE INVENTORY
+              REUSE STORAGE
+              IGNORE DELETE TRIGGERS
+              TRUNCATE TABLE INVENTORY
+              REUSE STORAGE
+              IGNORE DELETE TRIGGERS
```

Comment:
You copied 3 times the same variant. I'd like to see some statements with `RESTRICT WHEN DELETE TRIGGERS` and `DROP STORAGE` just to confirm in rdzSQL file that we get everything correctly.